backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/dto/UpdateNodeBoundsInput.java

```diff
@@ -31,11 +31,11 @@
 
     private UUID id;
 
-    private UUID editingContextId;
+    private String editingContextId;
 
-    private UUID representationId;
+    private String representationId;
 
-    private UUID diagramElementId;
+    private UUID nodeId;
```

Comment:
I don't mind this change but this API break is a bit outside of the scope of the issue

Reply:
Done